server/controllers/user.js

```diff
@@ -1,23 +1,179 @@
-const Todo = require('../models').Todo;
+import models from '../models';
+import jwt from 'jsonwebtoken';
+import config from '../config/config';
 
-module.exports = {
-  create(req, res) {
-    return Todo
-      .create({
-        title: req.body.title,
+const Users = models.Users;
+const Roles = models.Roles;
+
+ const isLoggedInUser = (userId, queryId) => {
+    if (parseInt(userId) === parseInt(queryId)) return true;
```

Comment:
It's advised to include a radix parameter when calling parseInt. Read more [here](http://eslint.org/docs/rules/radix). Look at the one you have in this [bin/www](https://github.com/andela-ieyo/docify/blob/develop/bin/www#L5)